### ISSUE TITLE
expose dist/ files in the package.json

### ISSUE THREAD

**Issue by gkatsev** [videojs/generator-videojs-plugin#41]
It would be nice if we could expose the `dist/` files in the package.json. browserify reads in either the `main` field or the `browser` field and we're already using the `main` field for the `es5/plugin.js` file and it makes sense for it to stay.
As part of [parcelify](https://www.npmjs.com/package/parcelify) a `style` property is often used to specify the styles for the package, so, we should use that.
There isn't currently a standard or a precedence in terms of exposing the "global" file in the package.json. `global` is probably too generic and would be confusing in the node-world.

**Comment by misteroneill** [comment]
As I understand it, you're suggesting something like:

```json
{
  "style": "dist/videojs-my-plugin.css",
  "some-unknown-name": "dist/videojs-my-plugin.js"
}
```

If it's not supported or used anywhere, what's the point? Should we not wait for a standard to emerge?

**Comment by misteroneill** [comment]
Some potential replacements for `"some-unknown-name"`: `"standalone"`, `"bundle"`, `"dist"`, `"javascript"`.

**Comment by gkatsev** [comment]
We can set the standard :P
Also, we can go with something like `videojs-plugin` to be vjs/our generator specific until such a standard does emerge.

**Comment by misteroneill** [comment]
Haha! What about:

```json
"videojs-plugin": {
  "style": "dist/videojs-foo.css",
  "script": "dist/videojs-foo.js"
},
```

**Comment by gkatsev** [comment]
That could work. I think that the `style` property is common enough that we could probably do both?

**Comment by misteroneill** [comment]
This will be fixed in #50, so tracking it there.

**Comment by gkatsev** [comment]
Also, take a look at https://github.com/npm/npm/issues/9459
